garrysmod/gamemodes/terrortown/gamemode/gamemsg.lua

```diff
@@ -245,9 +249,13 @@ local function MuteTeam(ply, cmd, args)
 
    local t = tonumber(args[1])
    ply.mute_team = t
-
-   local name = (t != 0) and team.GetName(t) or "None"
-   ply:ChatPrint(name .. " muted.")
+   if t == MUTE_ALL then
+   ply:ChatPrint("All muted.")
+   elseif t == MUTE_NONE or t == TEAM_UNASSIGNED then

```

Comment:
Afaik TEAM_UNASSIGNED is given to players who haven't been given a team yet. Even though you shouldn't ever have to mute TEAM_UNASSIGNED I don't see why group it with MUTE_NONE.